includes/class-mysql.php

```diff
@@ -76,18 +81,34 @@ function yourls_db_connect() {
  *     yourls_get_db()->do_stuff()
  *
  * @since  1.7.10
+ * @param string $context Optional context. Default: ''.
+ *   When provided, use a naming schema starting with a prefix describing the operation, followed by a short description:
+ *   - Prefix should be one of: "read-", "write-", or "other-".
+ *        * "read-" for operations that only read from the DB (eg get_keyword_infos)
+ *        * "write-" for operations that write to the DB (eg insert_link_in_db)
+ *        * "other-" for operations that do not fit in the other categories because they are not SQL queries (eg
+ *           when just storing info in memory, in the YDB object)
+ *   - The description should be lowercase, words separated with underscores, eg "insert_link_in_db".
```

Comment:
I would suggest validating the prefix somehow, or at least explicitly specifying how a `$context` that doesn't fit this naming scheme should be handled.